nh_eobs_mental_health/models/nh_clinical_sql.py

```diff
@@ -155,6 +155,53 @@ def get_ward_dashboard_reason_count(self, cr, reason):
     WHERE location.usage = 'ward'
     """
 
+    last_finished_pme_for_spell_skeleton = """
+    SELECT
+    activity.id AS id,
+    spell.activity_id AS spell_activity_id,
+    spell.id AS spell_id,
+    activity.date_terminated AS activity_date_terminated,
+    activity.state as activity_state
+    FROM nh_clinical_spell AS spell
+    JOIN (
+      SELECT DISTINCT ON (spell_activity_id) * FROM nh_activity
+      WHERE data_model = 'nh.clinical.patient_monitoring_exception'
+      AND state = ANY('{cancelled, completed}')
+      ORDER BY spell_activity_id, date_terminated DESC
+    ) AS activity
+    ON activity.spell_activity_id = spell.activity_id
+    """
+
+    def get_wardboard(self, interval):
+        """
+        Override wardboard SQL view to include acuity index
+        :param interval: Time interval used for recently transferred /
+        discharged
+        :return: SQL statement used in nh_eobs.init()
+        """
+        return self.wardboard_skeleton.replace(
+            'spell.move_date,',
+            'spell.move_date, '
+            'CASE '
+            'WHEN spell.obs_stop = \'t\' THEN \'ObsStop\' '
```

Comment:
I think Postgres accepts `TRUE` as well as `'t'` just in case you want to avoid those escaped apostrophes in future.